src/browser/main.coffee

```diff
@@ -78,6 +79,19 @@ setupCoffeeScript = ->
     js = CoffeeScript.compile(coffee, filename: filePath)
     module._compile(js, filePath)
 
+# Set Atom's home in process.env.ATOM_HOME
+setupConfigDirPath = ->
+  portablePath =
+    if process.platform == 'darwin'
+      process.resourcesPath.slice(0, -19) + '/.atom'
+    else
+      process.resourcesPath.slice(0, -10) + '/.atom'
```

Comment:
You'll want to use `path.join` to make sure the separator is correct for the platform